test.js

```diff
@@ -52,3 +52,14 @@ test('aliases', function (t) {
   const r = wayfarer()
   t.equal(r, r.emit)
 })
+
+test('.match()', function (t) {
+  t.plan(2)
+  const r = wayfarer()
+  r.on('/user/:id', function () {
+    t.fail('route should not be called with .match()')
```

Comment:
`coveralls` doesn't appear to be a fan of `t.fail` - https://coveralls.io/builds/2740508/source?filename=test.js :/